components/scream/src/physics/rrtmgp/mo_garand_atmos_io.cpp

```diff
@@ -0,0 +1,115 @@
+#include "mo_garand_atmos_io.h"
+#include "simple_netcdf.hpp"
+#include <netcdf.h>
+
+// Read in the data, then use only the first column, and copy it to all of the model columns
+// In the end, all model columns will be identical
+void read_atmos(std::string input_file, real2d &p_lay, real2d &t_lay, real2d &p_lev, real2d &t_lev,
+                GasConcs &gas_concs, real2d &col_dry, int ncol) {
+  simple_netcdf::SimpleNetCDF io;
+  io.open(input_file , NC_NOWRITE);
+
+  int nlay = io.getDimSize("lay");
+  int nlev = io.getDimSize("lev");
+
+  p_lay = real2d("p_lay",ncol,nlay);
+  t_lay = real2d("t_lay",ncol,nlay);
+  p_lev = real2d("p_lev",ncol,nlev);
+  t_lev = real2d("t_lev",ncol,nlev);
+
+  real2d tmp2d;
+  // p_lay
+  io.read(tmp2d,"p_lay");
+  // for (int ilay=1 ; ilay <= nlay ; ilay++) {
```

Comment:
Are we keeping these commented for loops in the code for any reason?

Reply:
This was external code that I duplicated solely for the purpose of using our own `simple_netcdf.hpp` library instead of YAKLs, so I'd prefer to keep the changes minimal to make it easier to diff against the original sources to see what we changed specifically. Matt likes the commented loops to help parse what the parallel_for is supposed to be doing.